src/components/Footer/Footer.scss

```diff
@@ -58,6 +58,15 @@
       flex-direction: row;
       margin-top: 120px;
     }
+    @media (max-width: 1450px) {
```

Comment:
I think this solution is too hacky, and the download button's position looks abit odd in my opinion.

<img width="1279" alt="Screen Shot 2021-07-03 at 3 20 11 PM" src="https://user-images.githubusercontent.com/32864116/124346562-2ac4e180-dc12-11eb-81f9-c0ad6bd43f2f.png">


Can you just decrease the `margin-right` of `FooterNavList` to `50px`? that would solve the issue

Reply:
Now good?